web/src/components/SearchFormGroup.tsx

```diff
@@ -71,27 +70,46 @@ export const SearchFormGroup: React.FC<SearchFormGroupProps> = ({
 
   const goalInput = (
     <InputGroup>
-      <InputGroupText isPlain>{t('Goal class for search:')}</InputGroupText>
       <InputGroupItem>
         <TextInput
           value={search.goal}
-          placeholder="domain:class"
+          placeholder="domain:classname"
           onChange={(e) => onChange({ ...search, goal: (e.target as HTMLInputElement).value })}
         />
       </InputGroupItem>
     </InputGroup>
   );
 
+  const help = (
+    <HelpPopover header={label}>
+      <Trans t={t}>
+        <DescriptionList>
+          <DescriptionListGroup>
+            <DescriptionListTerm>Distance</DescriptionListTerm>
+            <DescriptionListDescription>
+              Find all related items up to the specified distance.
```

Comment:
Translations should be scoped closer to the text and avoid HTML code, this will make them more maintainable.
```suggestion
              <Trans>Find all related items up to the specified distance.</Trans>
```